src/hotspot/share/gc/shenandoah/shenandoahPhaseTimings.hpp

```diff
@@ -57,6 +57,7 @@ class outputStream;
   f(conc_mark_roots,                                "Concurrent Mark Roots ")          \
   SHENANDOAH_PAR_PHASE_DO(conc_mark_roots,          "  CMR: ", f)                      \
   f(conc_mark,                                      "Concurrent Marking")              \
+  f(conc_mark_satb_flush_rendezvous,                "  SATB Flush Rendezvous")         \
```

Comment:
Could this be "Flush SATB" or "Flush SATB Handshakes" for consistency with https://github.com/openjdk/jdk/pull/20549?

Reply:
`Flush SATB` would be good, I think.

Reply:
will rename to "Flush STAB"